components/post-context/readme.md

```diff
@@ -0,0 +1,51 @@
+# `PostContext`
+
+The `PostContext` component allows you to customize the post object referenced by and of the components referencing the current post object. They are all prefixed with `Post`.
+
+For example this can be used to build a custom block that gets used inside the core query loop and accesses the passed in post id / post type of that to power the functionality of all the `Post` child components.
+
+## Usage
+
+```js
+import { PostContext, PostTitle } from '@10up/block-components';
+
+function BlockEdit() {
+
+    return (
+        <PostContext postId={42} postType={'page'} isEditable={true} >
+            <PostTitle tagName="h2" className="wp-block-example-hero__title" />
+        </PostContext>
+    )
+}
+```
+
+The `PostContext` component works great with the Core Query Loop block if you want / need to create a custom block to be used within the post template. Any block that gets used inside the query loop can access the current post id, post type, and query id via the block context. These values can then be used within the `<PostContext>` to make them available for any of the post level components nested within.
```

Comment:
Besides this example, in the description of the PR you mention `usesContext`, perhaps mention that and either do a quick demo or even link to Gutenberg training material (not sure if we have this covered)

Reply:
I added some more inline context here [fcb479e](https://github.com/10up/block-components/pull/178/commits/fcb479e97de9f9c793490553321ac6b9418e13bf)